### ISSUE TITLE
Possibility to turn off versioning

### ISSUE THREAD

**Issue by ruudk** [event-catalog/eventcatalog#533]
Is it possible to use Event Catalog without having to define versions? I want to have a catalog for our internal project. We don't have versions, so it looks weird to see 0.0.1 everywhere.

**Comment by boyney123** [comment]
Yeah not at the moment, versions are used under the hood for things, but I might be able to turn them off in the UI.

Would a flag in the eventcatalog.config.js file be enough? Stop all the version stuff being rendered to screen?

**Comment by ruudk** [comment]
That would be a pragmatic fix indeed.

Another one could be to have a default version, when not defined.

That would cleanup the definitions.

**Comment by otbe** [comment]
Want to +1 this for certain areas at least. Domains and Services ar enot versioned in that way in our org and its annoying to see the 0.0.1 everywhere :D

Same might be true for commands/queries depending on how and if we can use them.

**Comment by boyney123** [comment]
Cool, I might be able to remove them from the UI quite easily, but the routes/URLS not sure about yet....

First step will add a config option to disable for each resource (domain, service, events, commands).

**Comment by IsmaelMartinez** [comment]
I think being able to turn it on/off for visualisations/graphs (like the NodeGraph and visualisation) would be good. As even just having a couple of version in makes those visualisations loose their usefulness. 

Ideally, you should only see on them what the current selected version represents. That would make it easier for people to see when new events has been added/removed, etc. 

My 2 cents ;)

**Comment by gabrielcolson** [comment]
would be nice to be able to remove the versioning from the search bar as well! It adds a lot of noise, especially if you have no use for versioning at all...

**Comment by mrfelton** [comment]
We have a pretty large catalog (40 services, 1500 events). We don't currently do versioning and so the versioning is a bit of a distraction. However, the bigger problem for us is the sheer size of all of the additional output for the changelog (which we do't need since we aren't doing versioning).

Each generated static html page is ~3-4mb in size. With the changelog we get double the number of pages. Our fully built catalog comes in at about 14GB! Half of which seems to be the changelogs that don't serve a useful purpose for us,

To be honest, maybe my issue should be a separate bug? as it seems odd to me that each page would be 3-4 mb in size with a total dist output size of several GB.

**Comment by boyney123** [comment]
Hey @mrfelton thanks for the feedback.

Yeah the size thing sounds like a separate issue, happy to look into this. If you could provide any context you found, and ways to reproduce etc, that would be useful too.

Thanks!

**Comment by mrfelton** [comment]
Thanks - created an issue for the build size issue: https://github.com/event-catalog/eventcatalog/issues/1498
